dRep.spec

```diff
@@ -11,6 +11,6 @@ module dRep {
     /*
         This example function accepts any number of parameters and returns results in a KBaseReport
     */
-    funcdef run_dRep(mapping<string,UnspecifiedObject> params) returns (ReportResults output) authentication required;
+    funcdef dereplicate(mapping<string,UnspecifiedObject> params) returns (ReportResults output) authentication required;
```

Comment:
We should make a real type for the params here. While UnspecifiedObject is okay to get started, most people use this file to understand your api and it's useful to have the real types. Also, you can add documentation about all your params in the type.